fbw-a32nx/src/systems/instruments/src/NDv2/shared/map/WaypointLayer.ts

```diff
@@ -135,6 +137,27 @@ export class WaypointLayer implements MapLayer<NdSymbol> {
         context.font = '21px Ecam';
 
         PaintUtils.paintText(isColorLayer, context, x + 15, y + 17, symbol.ident, mainColor);
+
+        if (symbol.type & (NdSymbolTypeFlags.CourseReversalLeft | NdSymbolTypeFlags.CourseReversalRight)) {
+            this.paintCourseReversal(context, x, y, symbol);
+        }
+    }
+
+    private paintCourseReversal(context: CanvasRenderingContext2D, x: number, y: number, symbol: NdSymbol) {
+        const left = symbol.type & (NdSymbolTypeFlags.CourseReversalLeft);
+        const arcEnd = left ? -42 : 42;
+        context.strokeStyle = '#fff';
+        context.lineWidth = 1.75;
+        context.translate(x, y);
```

Comment:
The symbol should be rotated to the inbound course of the hold/procedure turn but I don't see that here. Was that lost with NDv2?

Reply:
Do you mean the rotation here, based on "direction"? https://github.com/flybywiresim/aircraft/blob/master/fbw-a32nx/src/systems/instruments/src/ND/elements/FlightPlan.tsx#L208C7-L208C27

Reply:
> Do you mean the rotation here, based on "direction"? https://github.com/flybywiresim/aircraft/blob/master/fbw-a32nx/src/systems/instruments/src/ND/elements/FlightPlan.tsx#L208C7-L208C27

Yes, exactly.

Reply:
Fixed in newest commit. Since it was connected I thought it didn't need to be a new PR.